api/v1/pinning.go

```diff
@@ -255,6 +255,17 @@ func (s *apiV1) handleAddPin(e echo.Context, u *util.User) error {
 		return err
 	}
 
+	// backwards compatibility to accept filename queryParam if name is not provided
```

Comment:
Let's make this change in handlers.go in `handleAddIpfs` instead of in the pinning code.

Reply:
i want this in `handleAddPin` so the `/add-ipfs` and `/pins` endpoints remain on the same codepath. and generally we should be trying to move handlers out of `handlers.go` and into `api/v1/<group>.go`

Reply:
handleAddIpfs is legacy and needs to be backward compatible. Adding this codepath here introduces new functionality to handleAddPin -- I don't think we should do that.

Reply:
Given that in api v2 handleAddIpfs will be removed anyway I think it makes sense to have the backward compat logic there instead of the pinning logic.

Reply:
i'd rather not do any backend change then and just let the frontend PR address the issue of a blank name in UI. gonna close this out